src/components/calendar-date/calendar-date.module.scss

```diff
@@ -0,0 +1,46 @@
+.container {
+    width: 35px;
+    height: 35px;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    cursor: pointer;
+    border-radius: 50%;
+
+}
+
+.date {
+    color: white;
```

Comment:
Should use css vars.

https://github.com/codux-demos/bookings-starter/blob/main/src/styles/theme.module.scss